oshi-core/src/main/java/oshi/software/os/mac/MacOperatingSystem.java

```diff
@@ -154,6 +157,18 @@ private String parseCodeName() {
                 return "Cheetah";
             default:
             }
+            */
+            final String OSHI_VERSIONS_PROPERTIES = "oshi.versions.properties";
+            Properties verProps = FileUtil.readPropertiesFromFilename(OSHI_VERSIONS_PROPERTIES);
+
+            String major = String.valueOf(this.major);
+            String minor = String.valueOf(this.minor);
+            String version_number = major+"."+minor;
```

Comment:
You don't need to do the `String.valueof()` before this, as string concatenation will convert to strings anyway.   
 Just use `this.major + "." + this.minor`.  (Also using the same name for a local variable as a class/instance variable isn't a good idea.).   Also, checkstyle is complaining that you didn't leave whitespace before and after the `+`.